lib/OpenQA/Worker/Settings.pm

```diff
@@ -52,6 +52,7 @@ sub new {
 
     # read global settings from environment variables
     $global_settings{LOG_DIR} = $ENV{OPENQA_WORKER_LOGDIR} if $ENV{OPENQA_WORKER_LOGDIR};
+    $global_settings{TERMINATE_AFTER_JOBS_DONE} //= $ENV{OPENQA_WORKER_TERMINATE_AFTER_JOBS_DONE};
```

Comment:
We should probably have an uncommented example in the default `workers.ini`?

Reply:
has been done

Reply:
I have added a comment to `workers.ini`.